challenge5/index.html

```diff
@@ -0,0 +1,292 @@
+  <script>
+    tailwind.config = {
+      theme: {
+        extend: {
+          colors: {
+            primary_color: "#FFFFFF",
+            secondary_color: "#458FF6",
+            top_gradient: "#5A98F2",
+            bottom_gradient: "#67C3F3",
+            background: "#FAF9F6",
+          },
+          lineHeight: {
+            12: "60px",
+          },
+          rotate: {
+            160: "160deg",
+          },
+          padding: {
+            11.5: "46.5px",
+            15: "60px",
+          },
+          fontFamily: {
+            sans: ['Mulish', 'sans-serif'],
+          },
+          width: {
+            65: "259px",
+            button: "199px",
+          },
+          fontSize: {
+            button: ["18px", "60px"],
+          },
+        },
+      },
+    };
+  </script>
+</head>
+
+<body class="bg-background z-0 font-sans">
+  <header class="flex flex-row px-8 pt-2 pb-15 justify-between">
+    <div class="flex flex-row">
+      <img src="./assets/icons/logo.svg" alt="Logo" />
+    </div>
+    <div>
+      <div class="hidden md:flex">
+        <ul class="flex flex-row gap-x-5 pl-10">
+          <li class="font-medium">Home</li>
+          <li class="text-gray-400 font-medium">Find a doctor</li>
+          <li class="text-gray-400 font-medium">Apps</li>
+          <li class="text-gray-400 font-medium">Testimonials</li>
+          <li class="text-gray-400 font-medium">About us</li>
+        </ul>
+      </div>
+      <div class="flex md:hidden">
+        <img src="./assets/icons/List.svg" alt="Menu" />
+      </div>
+    </div>
+  </header>
+  <main class="flex flex-col overflow-hidden">
+    <section class="relative flex flex-col md:flex-row pl-9 pr-11.5 justify-center grow">
+      <div class="flex flex-col pr-10 pb-9 gap-y-6 self-center max-w-md">
+        <h1 class="pr-12 text-3xl font-bold">
+          Virtual healthcare for you
+        </h1>
+        <p class="text-lg font-light text-gray-400">
+          Trafalgar provides progressive, and affordable healthcare,
+          accessible on mobile and online for everyone
+        </p>
+        <button class="bg-secondary_color text-white rounded-full text-button px-10 w-button hover:opacity-80">
+          Consult today
+        </button>
+      </div>
+      <div class="flex flex-auto md:w-2/3 max-w-xl">
+        <img src="./assets/imgs/imgSection1.svg" alt="Illustration" class="flex flex-auto md:w-2/3" />
+      </div>
+    </section>
+
+    <section class="relative flex flex-col px-15 pt-20 self-center justify-center">
+      <img src="./assets/imgs/bgImg.svg" alt="Background Image" class="absolute left-0 transform -rotate-160 ">
+      
+      <div class="flex flex-col md:w-2/3 gap-y-6 self-center relative z-10">
+        <h2 class="text-3xl font-bold text-center">
+          Our services
+        </h2>
+        <div class="self-center hidden md:inline">
+          <img src="./assets/icons/Rectangle.svg" alt="Vector" />
+        </div>
```

Comment:
Same here (and all other decorative images).

Also, no need to change it, but this could have been implemented with a div with a background color instead of using an SVG, it wouldn't be a lot of work